includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -1,377 +1,610 @@
 					],
-					'href' => 'https://www.instagram.com/getfandom/'
-				]
-			]
-		],
-		'wikia' => [
-			'header' => [
-				'type' => 'line-image',
-				'image' => 'company-logo-wikia',
-				'title' => [
-					'type' => 'translatable-text',
-					'key' => 'global-footer-wikia-header'
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-footer-wikia-community-link-support'
+						],
+						'href' => $this->getHref( 'support' )
+					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-footer-wikia-community-link-fan-contributor-program'
+						],
+						'href' => $this->getHref( 'fan-contributor' )
+					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-footer-wikia-community-link-wam-score'
+						],
+						'href' => $this->getHref( 'wam' )
+					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-footer-wikia-community-link-help'
+						],
+						'href' => $this->getHref( 'help' )
+					]
 				]
-			]
-		],
-		'company_overview' => [
-			'header' => [
-				'type' => 'line-text',
-				'title' => [
+			],
+			'create_wiki' => [
+				'description' => [
 					'type' => 'translatable-text',
-					'key' => 'global-footer-wikia-company-overview-header'
+					'key' => 'global-footer-wikia-create-wiki-description'
+				],
+				'links' => [
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-footer-wikia-create-wiki-link-start-wikia'
+						],
+						'href' => $this->getHref( 'create-new-wiki' )
+					]
 				]
 			],
-			'links' => [
-				[
-					'type' => 'link-text',
+			'community_apps' => [
+				'header' => [
+					'type' => 'line-text',
 					'title' => [
 						'type' => 'translatable-text',
-						'key' => 'global-footer-wikia-company-overview-link-about'
-					],
-					'href' => 'http://www.wikia.com/About'
+						'key' => 'global-footer-wikia-community-apps-header'
+					]
 				],
-				[
-					'type' => 'link-text',
-					'title' => [
-						'type' => 'translatable-text',
-						'key' => 'global-footer-wikia-company-overview-link-careers'
-					],
-					'href' => 'https://careers.wikia.com/'
+				'description' => [
+					'type' => 'translatable-text',
+					'key' => 'global-footer-wikia-community-apps-description'
 				],
-				[
-					'type' => 'link-text',
-					'title' => [
-						'type' => 'translatable-text',
-						'key' => 'global-footer-wikia-company-overview-link-press'
+				'links' => [
+					[
+						'type' => 'link-image',
+						'image' => 'company-store-appstore',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-footer-wikia-community-apps-link-app-store'
+						],
+						'href' => 'https://itunes.apple.com/pl/developer/wikia-inc./id422467077'
```

Comment:
Great question, I just moved it and didn't notice it. Probably it's not the best link for all languages...